keyboards/lets_split/rev1/rules.mk

```diff
@@ -1 +1 @@
-BACKLIGHT_ENABLE = no
\ No newline at end of file
+BACKLIGHT_ENABLE = no
```

Comment:
This file could be zero-length; Backlight is already disabled at keyboard level. Same for `keyboards/lets_split/rev2/rules.mk` also (which is unedited by this PR).